### PR TITLE
437 date display component

### PR DESCRIPTION
**Pull request by Chrischuck**
#437

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -7,7 +7,7 @@
     "ios": "react-native run-ios",
     "start": "react-native start",
     "clean-start": "npx react-native clean-project-auto && npx pod-install && npm start",
-    "test": "jest",
+    "test": "TZ=UTC jest",
```
Comment (kueda) [inline package.json:10]: Cool, didn't know you could do this.

File: src/components/SharedComponents/DateDisplay.js [hunk 2]
```diff
@@ -0,0 +1,35 @@
...
+  dateTime
+}: Props ): React.Node => {
+  const { t } = useTranslation( );
+  const displayTime = ( ) => {
+    if ( dateTime ) {
+      return formatObsListTime( dateTime );
+    }
+    return t( "no time given" );
+  };
+  console.log( displayTime() );
```
Comment (kueda) [inline src/components/SharedComponents/DateDisplay.js:24]: Please remove.

**Review by kueda — CHANGES_REQUESTED** [review]
Looking good, only minor changes needed. Date formatting needs to be handled a little differently but that wasn't in the issue so I punted that to #448. FWIW, I'm also getting some `Warning: An update to DateDisplay inside a test was not wrapped in act(...).` errors when running tests, though I'm not 100% sure why. Would be nice to fix that, but I don't think it needs to block.

**Comment by Chrischuck** [comment]
@kueda Looks like it has to do with the `useTranslation` hook. After removing it and running tests it seems to disappear. I can take a look into it as another issue.

**Review by kueda — APPROVED** [review]

**Pull request merged**